CMakeLists.txt

```diff
@@ -67,55 +68,76 @@ set(drake_dependencies cmake eigen lcm libbot bullet octomap snopt gurobi)
 # download information, in alphabetical order
 set(avl_GIT_REPOSITORY https://github.com/RobotLocomotion/avl.git)
 set(avl_GIT_TAG 9b927f90619460b29f7454c714369f65bc4536a1)
+set(avl_IS_PUBLIC TRUE)
 set(bertini_GIT_REPOSITORY https://github.com/RobotLocomotion/bertini.git)
 set(bertini_GIT_TAG 3ae3e3ad3534acb4b6f7a4c3c22728b493beaa80)
+set(bertini_IS_PUBLIC FALSE)
 set(bullet_GIT_REPOSITORY https://github.com/RobotLocomotion/bullet-pod.git)
 set(bullet_GIT_TAG 78c36cd116becf8417334f98bfcfd8d1300b03e0)
+set(bullet_IS_PUBLIC TRUE)
 set(cmake_GIT_REPOSITORY https://github.com/RobotLocomotion/cmake.git)
 set(cmake_GIT_TAG 5ea95a6bb936c5fb8635b671a3355d86a882f5f7)
 set(cmake_GIT_CLONE_DIR "${PROJECT_SOURCE_DIR}/drake/cmake")
 set(cmake_BUILD_COMMAND "")
+set(cmake_IS_PUBLIC TRUE)
 set(director_GIT_REPOSITORY https://github.com/RobotLocomotion/director.git)
 set(director_GIT_TAG 9c2bb9de70a0945d05eb0a4a7d13c0a91afa74a3)
 set(director_SOURCE_DIR ${PROJECT_SOURCE_DIR}/externals/director/distro/pods/drake-distro)
+set(director_IS_PUBLIC TRUE)
 set(eigen_GIT_REPOSITORY https://github.com/RobotLocomotion/eigen-pod.git)
 set(eigen_GIT_TAG 9fdb567778561415ca7032a64b8431484900798e)
 set(eigen_IS_CMAKE_POD TRUE)
+set(eigen_IS_PUBLIC TRUE)
 set(gtk_GIT_REPOSITORY https://github.com/RobotLocomotion/gtk-pod.git)
 set(gtk_GIT_TAG dc9bb719ee771a2ae9c3a392796c42ccad5bca54)
+set(gtk_IS_PUBLIC TRUE)
 set(gloptipoly_GIT_REPOSITORY https://github.com/RobotLocomotion/gloptipoly3.git)  # todo: rename that repo
 set(gloptipoly_GIT_TAG c9e796e99c584ecf78317348c4162a2ed4121156)
+set(gloptipoly_IS_PUBLIC FALSE)
 set(gurobi_GIT_REPOSITORY https://github.com/RobotLocomotion/gurobi.git)
 set(gurobi_GIT_TAG beb0a84a1bcca1eb81a1b87d9fa140f2134d97e9)
+set(gurobi_IS_PUBLIC TRUE)
```

Comment:
maybe keep it for this week, but i’m thinking we might want yet another option for PUBLIC_OR_FREE_FOR_ACADEMICS or something?

> On Sep 14, 2015, at 8:54 PM, Robin Deits <notifications@github.com> wrote:
> 
> In CMakeLists.txt <https://github.com/RobotLocomotion/drake/pull/1343#discussion_r39464890>:
> 
> >  set(gurobi_GIT_REPOSITORY https://github.com/RobotLocomotion/gurobi.git)
> >  set(gurobi_GIT_TAG beb0a84a1bcca1eb81a1b87d9fa140f2134d97e9)
> > +set(gurobi_IS_PUBLIC TRUE)
> Hm. True. But it's also a pretty important dependency, since without it we can't run the atlas simulations. I'm ambivalent.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/RobotLocomotion/drake/pull/1343/files#r39464890>.
>